sdk/identity/azure-identity/src/main/java/com/azure/identity/SharedTokenCacheCredential.java

```diff
@@ -86,22 +85,18 @@
 
                 if (accounts.size() == 0) {
                     if (username == null) {
-                        return Mono.error(new RuntimeException("No accounts were discovered in the shared token cache."
-                            + " To fix, authenticate through tooling supporting azure developer sign on."));
+                        return Mono.error(new RuntimeException("SharedTokenCacheCredential authentication unavailable. No accounts were discovered in the cache."));
```

Comment:
This is removing the instructions for the user as to how they'll resolve/fix the issue. We should keep those instructions so users are updated about how to resolve the issue.

Reply:
I removed these more specific error messages from the shared token credential. They aren't always as helpful, especially in this case where "To fix, authenticate through tooling supporting azure developer sign on." is very vague. The end goal as we add more specific credential types for authenticating with specific tools that they will provide error messaging which can be much more specific.